### PR TITLE
NEUBURG-23: Update eslint config

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

**Comment by michael-markl** [comment]
No refactorings necessary?  :D

**Comment by maxammann** [comment]
I need to add WIP: 
:joy:

**Review by NKlug — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -33,19 +33,25 @@
...
-    "eslint-config-standard": "^11.0.0",
+    "eslint": "~4.19.1",
+    "eslint-config-standard": "~11.0.0",
+    "eslint-config-standard-react": "~6.0.0",
     "eslint-plugin-flowtype": "^2.46.3",
-    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-jest": "~21.15.0",
+    "eslint-plugin-prefer-object-spread": "~1.2.1",
+    "eslint-plugin-react": "~7.7.0",
+    "eslint-plugin-standard": "~3.0.1",
```
Comment (michael-markl) [inline package.json]: when do you use `~` and when `^` ?

File: src/services/OfferService.js [hunk 2]
```diff
@@ -7,7 +7,11 @@ import createToken from '../utils/createToken'
 import MailService from './MailService'
 import forms from '../models/forms'
 
-const MILLISECONDS_IN_A_DAY = 1000 * 60 * 60 * 24
+const MS_IN_S = 1000
+const S_IN_MIN = 60
+const MIN_IN_H = 60
+const H_IN_D = 60
+const MILLISECONDS_IN_A_DAY = MS_IN_S * S_IN_MIN * MIN_IN_H * H_IN_D
```
Comment (michael-markl) [inline src/services/OfferService.js:21]: Imo this is so much uglier than it could be :D but oh well^^

**Review by michael-markl — APPROVED** [review]

File: src/services/OfferService.js (hunk 2 above)
Comment (maxammann) [reply src/services/OfferService.js:21]: We can discuss the linting rules, BUT: This makes clear that a library for time should be used here which provides a type for a duration.

File: package.json (hunk 1 above)
Comment (maxammann) [reply package.json]: I switched back to the node default ^ in the webapp. No need to be consistent. I makes no diference when you execute: `yarn upgrade --latest`.

http://fredkschott.com/post/2014/02/npm-no-longer-defaults-to-tildes/

File: src/services/OfferService.js (hunk 2 above)
Comment (michael-markl) [reply src/services/OfferService.js:21]: I don't think `1000*60*60*24` exceeds our mental capacities to a limit where we need to introduce a library...
I mean when we're in backend, I don't really care, if we use hundreds of libs, but once we're speaking of frontend, we should really seek to reduce the amount of external librarys... especially if the environment already (sort of) supports the features out of the box, but maybe without a perfect api...

**Pull request merged**